RxMediaPicker/RxMediaPicker.swift

```diff
@@ -92,79 +93,78 @@ public enum RxMediaPickerError: Error {
                 picker.cameraFlashMode = flashMode
             }
             
-            self.delegate?.present(picker: picker)
+            self.presentPicker(picker)
             
             return Disposables.create()
-        })
+        }
     }
     
     open func selectImage(source: UIImagePickerControllerSourceType = .photoLibrary, editable: Bool = false) -> Observable<(UIImage, UIImage?)> {
         
-        return Observable.create { observer in
+        return Observable.create { [unowned self] observer in
             self.currentAction = RxMediaPickerAction.photo(observer: observer)
             
             let picker = UIImagePickerController()
             picker.sourceType = source
             picker.allowsEditing = editable
             picker.delegate = self
             
-            self.delegate?.present(picker: picker)
+            self.presentPicker(picker)
             
             return Disposables.create()
         }
     }
     
     func processPhoto(info: [String : AnyObject], observer: AnyObserver<(UIImage, UIImage?)>) {
-        if let image = info[UIImagePickerControllerOriginalImage] as? UIImage {
-            let editedImage: UIImage? = info[UIImagePickerControllerEditedImage] as? UIImage
-            observer.on(.next(image, editedImage))
-            observer.on(.completed)
-        } else {
+        guard let image = info[UIImagePickerControllerOriginalImage] as? UIImage,
+              let editedImage = info[UIImagePickerControllerEditedImage] as? UIImage else {
             observer.on(.error(RxMediaPickerError.generalError))
```

Comment:
Uhm I guess it's a matter of preference, Do you know of any other RxSwiftCommunity repo using this approach? Separating the return of the guard makes it a bit more readable in my opinion and also doesn't make the developer expect something being returned from that when reading. (I don't have a super strong opinion here I would just want to see that directly doing a return in a guard is being used in this context in other places)